templates/dashboard/dashboard_spa.html

```diff
@@ -0,0 +1,318 @@
+            <input type="search"
+                   name="query"
+                   v-model="searchString"
+                   placeholder="Enter to search for alias"
+                   class="form-control shadow mr-2"
+                   @keyup.enter="loadAliases">
+          </div>
+          <div v-if="searchString || filter" class="ml-2">
+            <a @click="resetFilter()" class="btn btn-outline-secondary">Reset</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- END Controls: buttons & search -->
+    <!-- Alias list -->
+    <div id="alias-list" class="row">
+      <div v-for="(alias, index) in aliasesArray"
+           :key="alias.id"
+           class="col-12 col-md-6">
+        <!-- TODO highlight newly created alias -->
+        <div class="card p-4 shadow-sm"
+             :class="{ 'alias-disabled': !alias.enabled }">
+          <div class="d-flex justify-space-between">
+            <div class="d-flex">
+              <h5 class="clipboard mb-0 text-break d-inline-block cursor"
+                  :data-toggle="alias.enabled ? 'tooltip' : null"
+                  title="Click to copy"
+                  :data-clipboard-text="alias.enabled ? alias.email : null">
+                [[ alias.email ]]
+              </h5>
+              <span v-if="alias.pinned"
+                    data-toggle="tooltip"
+                    title="Click to unpin"
+                    @click="handlePin(alias)"
+                    class="cursor ml-2">
+                <i class="fa fa-thumb-tack"></i>
+              </span>
+            </div>
+            <div class="col text-right">
+              <label class="custom-switch cursor"
+                     data-toggle="tooltip"
+                     :title="alias.enabled ? 'Disable alias - Stop receiving emails sent to this alias' : 'Enable alias - Start receiving emails sent to this alias'">
+                <input type="checkbox"
+                       class="enable-disable-alias custom-switch-input"
+                       v-model="alias.enabled"
+                       @change="toggleAlias(alias)">
+                <span class="custom-switch-indicator"></span>
+              </label>
+            </div>
+          </div>
+          <!-- Email Activity -->
+          <div v-if="!alias.enabled" class="small-text mb-4">
+            Alias is disabled, you will not receive any email from this alias.
+          </div>
+          <div v-else-if="alias.latest_activity" class="small-text mb-4">
+            [[ alias.latest_activity.contact.email ]]
+            <span v-if="alias.latest_activity.action === 'bounched'">
+              <i class="fa fa-exclamation-triangle mr-2"></i> Cannot be forwarded to your mailbox.
+            </span>
+            <span v-if="alias.latest_activity.action === 'block'">
+              <i class="fa fa-ban mr-2"></i> Blocked.
+            </span>
+            <!-- TODO improve date formatting -->
+            Created [[ alias.creation_date ]].
```

Comment:
I think we can use this code snippet https://stackoverflow.com/a/6109105 here to have a relative date time.